docs/cookbook/creating_landing_page_layouts_(enterprise).md

```diff
@@ -1,114 +1,93 @@
-        zones:
-            first:
-                name: First zone
-
-    1_2:
-        identifier: 1_2
-        name: Two zones in columns, narrow left, wide right
-        description: Two zones in columns, narrow left, wide right
-        thumbnail: '/bundles/ezstudiodemo/images/layouts/1_2.png'
-        template: eZStudioDemoBundle:layouts:1_2.html.twig
-        zones:
-            first:
-                name: First zone
-            second:
-                name: Second zone
-
-    1_2__1:   
-        identifier: 1_2__1
-        name: Three zones, two columns, narrow left, wide right in first row and one row
-        description: Three zones, two columns, narrow left, wide right in first row and one row
-        thumbnail: '/bundles/ezstudiodemo/images/layouts/1_2__1.png'
-        template: eZStudioDemoBundle:layouts:1_2__1.html.twig
-        zones:
-            first:
-                name: First zone
-            second:
-                name: Second zone
-            third:
-                name: Third zone
+ez_systems_landing_page_field_type:
+    layouts:
+        sidebar:
+            identifier: sidebar
+            name: Right sidebar
+            description: Main section with sidebar on the right
+            thumbnail: assets/images/layouts/sidebar.png
+            template: AppBundle:layouts:sidebar.html.twig
+            zones:
+                first:
+                    name: First zone
+                second:
+                    name: Second zone
 ```
 
-The following parameters need to be included in the settings of the default\_layouts.yml file:
+The following parameters need to be included in the settings of the configuration:
 
 |Parameter|Type|Description|Required|
 |------|------|------|------|
 |layouts|string|Layout config root|Yes|
 |number|string|Unique key of the layout|Yes|
-|{ID}/identifier|string|ID of the Layout|Yes|
-|{ID}/name|string|Name of the Layout|Yes|
-|{ID}/description|string|Description of Layout|Yes|
+|{ID}/identifier|string|ID of the layout|Yes|
+|{ID}/name|string|Name of the layout|Yes|
+|{ID}/description|string|Description of layout|Yes|
 |{ID}/thumbnail|string|<path> to thumbnail image|Yes|
-|{ID}/template|string|<path> to template View For example: `eZStudioDemoBundle:layouts:1.html.twig`</br>`<bundle>:<directory>:<file name>`|Yes|
+|{ID}/template|string|<path> to template View For example: `AppBundle:layouts:sidebar.html.twig`</br>`<bundle>:<directory>:<file name>`|Yes|
 |{ID}/zones|string|Collection of zones|Yes|
 |{ID}/{zone}/zone_id|string|ID of the zone|Yes|
 |{ID}/{zone}/name|string|Zone name|Yes|
+
+## Layout template
+
+A layout template will include all zones the layout contains. The zone container must have a `data-studio-zones-container` attribute.
+
+A Zone is a container for blocks. Each zone must have a `data-studio-zone` attribute.
+The best way to display blocks in the zone is to iterate over a blocks array and render the blocks in a loop.
```

Comment:
I would like to have more information: what is in the zones[0] and zones[1]?
In the example, the <div> code is the same for both. 
What do I have to prepare in advance to make this work?

Reply:
`<div class="landing-page__block block_{{ block.type }}">`

I know that it's the class that is changing, but I would like to have a precision about this.

Reply:
I had improved the code block below, but of course github didn't know to mark this comment as resolved :)

Here are the explanations:

https://github.com/ezsystems/developer-documentation/pull/142/files#diff-e10858ab1dbbe45ae8ff4642ff4c28adR63
https://github.com/ezsystems/developer-documentation/pull/142/files#diff-e10858ab1dbbe45ae8ff4642ff4c28adR68
https://github.com/ezsystems/developer-documentation/pull/142/files#diff-e10858ab1dbbe45ae8ff4642ff4c28adR81